### ISSUE TITLE
Complex API of initiation sail+spin+lucene

### ISSUE THREAD

**Issue by jgrzebyta** [eclipse/rdf4j#739]
Related to #220  and #584 

Example of SailRepository with Lucene and SpinSail initiation showed in class `org.eclipse.rdf4j.sail.lucene.AbstractLuceneSailSpinTest.setUp` is complex.

I guess it should be simplified and after that it is worth to add layout file for new store in `rdf4j-workbench`.

Also Lucene index is not built auto-magically. It might be built after commit (?).

**Comment by jgrzebyta** [comment]
Closed because it is related to #641 .

**Comment by jgrzebyta** [comment]
Reopened because it is sub-task of #641.

**Comment by jgrzebyta** [comment]
Created branch at github.com/jgrzebyta/rdf4j/  called *issues/#739-spin-lucene* .

Because of #756 there is problem with:
```
simpleTest(org.eclipse.rdf4j.sail.elasticsearch.ElasticsearchSailSpinTest)  Time elapsed: 0.114 sec  <<< ERROR!
java.lang.NoClassDefFoundError: org/apache/lucene/index/LeafReader
	at org.eclipse.rdf4j.sail.elasticsearch.ElasticsearchSailSpinTest.setUp(ElasticsearchSailSpinTest.java:33)
Caused by: java.lang.ClassNotFoundException: org.apache.lucene.index.LeafReader
	at org.eclipse.rdf4j.sail.elasticsearch.ElasticsearchSailSpinTest.setUp(ElasticsearchSailSpinTest.java:33)
```

**Comment by jgrzebyta** [comment]
Merged in 86aa4d4a3ed7e2a6cfb41736d0f4918c8a2784f7
